rllib/algorithms/dqn/dqn_rainbow_rl_module.py

```diff
@@ -29,6 +31,11 @@
 class DQNRainbowRLModule(RLModule, InferenceOnlyAPI, TargetNetworkAPI):
     @override(RLModule)
     def setup(self):
+        # TODO (sven): Temporary fix (until we have figured out the final config
+        #  architecture for default models). If self.model_config is a dict (which
+        #  it should always be, make sure to merge it with the default config).
+        self.model_config = dataclasses.asdict(DefaultModelConfig()) | self.model_config
```

Comment:
Does this mean that the user needs to do this everytime she defines a new module? We do here actually the same we did before with the `AlgorithmConfig.model_config` property.

Reply:
> Does this mean that the user needs to do this everytime she defines a new module? We do here actually the same we did before with the `AlgorithmConfig.model_config` property

Nope, this is only(!) needed for RLlib's default models. See my other answer here. Users are completely free to request any keys they need in their custom model classes and then - of course - have to provide these keys in the `model_config` dict they pass into the `RLModuleSpec`. Users can also decide to use default values inside their `RLModule` classes for certain keys.